store/store.go

```diff
@@ -406,6 +426,7 @@ const (
 	snapActionEndpPath = "v2/snaps/refresh"
 	snapInfoEndpPath   = "v2/snaps/info"
 	cohortsEndpPath    = "v2/cohorts"
+	findEndpPath       = "v2/snaps/find"
```

Comment:
thanks, should we bump the version we check to 20 as well?

Reply:
Yes, I just pushed it.